src/customers/components/CustomerDetailsPage/CustomerDetailsPage.tsx

```diff
@@ -105,12 +109,17 @@ const CustomerDetailsPage: React.FC<CustomerDetailsPageProps> = ({
                   onChange={change}
                 />
                 <CardSpacer />
-                <CustomerOrders
-                  orders={mapEdgesToItems(customer?.orders)}
-                  onViewAllOrdersClick={onViewAllOrdersClick}
-                  onRowClick={onRowClick}
-                />
-                <CardSpacer />
+                <RequirePermissions
+                  userPermissions={userPermissions}
```

Comment:
I'm not sure why we pass every time the same `userPermissions` to `RequirePermissions` component and why can't we use `useUserPermissions` simply inside it and only have to pass `requiredPermissions` prop?

Reply:
@orzechdev Actually that's a valid idea, I'll refactor it 👍